### PR TITLE
Make `random()` iterable

### LINKED ISSUE #7: Add generator method
How we can using it in for loop?

### PR DESCRIPTION
**Pull request by Richienb**
Request for comment

Fixes #7

### CODE REVIEW

**Comment by sindresorhus** [comment]
Good idea 👍

File: assert-in-range.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import chalk from 'chalk';
+import inRange from 'in-range';
+
+export default function assertInRange(t, value, {start = 0, end}) {
```
Comment (Richienb) [inline assert-in-range.js:4]: https://github.com/avajs/ava/issues/3314

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -11,7 +11,6 @@ jobs:
       matrix:
         node-version:
           - 14
-          - 12
```
Comment (Richienb) [inline .github/workflows/main.yml:14]: Updating tsd past the version that supports Node.js 12 fixes a problem, which for me, causes it to find and display type errors in `@types/node`

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by sindresorhus** [comment]
@Richienb Ready to release?

**Pull request merged**